### ISSUE TITLE
[Feature Request] Support Markdown syntax on descriptions and titles

### ISSUE THREAD

**Issue by feliperoos** [likec4/likec4#1124]
First of all, thanks for providing such a tool for the community. I appreciate the effort being put into keeping the tool fresh and updated.

One thing I'd love to see is the ability to add MD syntax to have elements in bold, italicized, underlined, etc, to provide emphasis, especially on relationships.

Some examples would include 
```
{
  service = element {
    notation 'Service'
  }
}
model {
  a = service 'Service A'
  b = service 'Service B'
  a -> b 'A calls B __only once__' {
    notes '
       This relationship is being **__deprecated__** due to:
        1. Not important
        2. Causing problems
        3. Wrong place
    ' 
  }
}
```

Hope this helps. Thank you!

**Comment by davydkov** [comment]
I'm working on it 👍🏻

**Comment by josdeweger** [comment]
@davydkov can you share what you have so far, and your thoughts on how to implement this?

**Comment by kieronlanning** [comment]
I'd love to see this one too.

**Comment by anujkumar-df** [comment]
Eagerly waiting for this. I am using C4 as the primary source of project documentation, and would love this feature to extend the diagrams even further.

**Comment by josdeweger** [comment]
See also https://github.com/likec4/likec4/discussions/1458

**Comment by davydkov** [comment]
Apparently my memory took a coffee break. This one completely slipped out.

**Comment by 304NotModified** [comment]
Would be great to see this feature!

**Comment by AndersHogqvist** [comment]
Yes indeed, this would be a very nice feature! If possible I'd like to see table support as well.

**Comment by vincentfree** [comment]
Maybe this could be extended to dynamic views, the notes could benefit from some styling options which can be provided by markdown support
